SHOE/Source/AssetManager.cpp

```diff
@@ -1669,20 +1681,10 @@ void AssetManager::InitializeTerrainMaterials() {
 
 		CreateWICTextureFromFile(device.Get(), context.Get(), wPath.c_str(), nullptr, defaultBlendMap.GetAddressOf());
 
-		// Serialize the filename if it's in the right folder
-		std::string baseFilename = "";
-		std::string assetPathStr = "Assets\\Textures";
-		std::string pathBufString = std::string(pathBuf);
-		size_t dirPos = pathBufString.find(assetPathStr);
-		if (dirPos != std::string::npos) {
-			// File is in the assets folder
-			baseFilename = "t";
-			baseFilename += pathBufString.substr(dirPos + assetPathStr.size() + 1);
-		}
-		else {
-			baseFilename = "f";
-			baseFilename += pathBufString;
-		}
+		std::string baseFilename;
```

Comment:
Can be simplified to
std::string baseFilename = SerializeFileName("Assets\\Textures\\", pathBuf);
or even just moved into SetFileNameKey in the next line